Dixit_Logic/Classes/DixitGame.cs

```diff
@@ -170,7 +175,27 @@ public void NewGuess(IPlayer player, ICard card)
         /// <param name="card">The card what will be put by "player".</param>
         public void PutCard(IPlayer player, ICard card)
         {
-            throw new NotImplementedException();
+            if(_actGameState.RoundStatus == PhaseStatus.Putting)
+            {
+                IDeck playerHand;
+
+                if(player != null && _actGameState.Hands.TryGetValue(player, out playerHand))
+                {
+                    ICard selectedCard = ((Hand)playerHand).GetCard(card);
+                    if (selectedCard != null)
+                    {
+                        _actGameState.BoardDeck.AddCard(selectedCard);
```

Comment:
I will add a player information to the cards what will indicate the player who drawed them.